orca-6/Dockerfile

```diff
@@ -6,11 +6,11 @@ RUN brew update \
 && brew uninstall ruby \
 && brew uninstall libbigwig \
 && brew uninstall libdivsufsort \
+&& brew update quicktree \
```

Comment:
@lcoombe This line should be `brew upgrade quicktree`

Reply:
Yes sorry for the mistake! Fixed in #92

Reply:
No worries at all